lib/inline-imports.js

```diff
@@ -86,7 +86,48 @@ function inlineImports(fileContents, options = {}, visited = new Set()) {
   return result.join('\n');
 }
 
+function getFilename(filePath, options = {}) {
+  const { resolveImport, resolveOptions = {} } = options;
+
+  const { basedir } = resolveOptions;
+  if (typeof basedir !== 'string') {
+    throw new Error('inlineImports requires options.resolverOptions.basedir be set');
+  }
+  const resolve = typeof resolveImport === 'function' ? resolveImport : require('resolve').sync;
+  return resolve(filePath, resolveOptions);
+}
+
+function getFileContents(filePath, options = {}) {
+  const { fs } = options;
+  const resolvedFs = typeof fs === 'function' ? fs : require('fs');
```

Comment:
as above i'd like to use a common utility rather than require every use of `fs` to check for a custom `fs` option